apps/meeting/src/containers/MeetingForm/index.tsx

```diff
@@ -25,7 +26,14 @@ import DevicePermissionPrompt from '../DevicePermissionPrompt';
 import RegionSelection from './RegionSelection';
 import { fetchMeeting, createGetAttendeeCallback } from '../../utils/api';
 import { useAppState } from '../../providers/AppStateProvider';
-import { MeetingMode } from '../../types';
+import { MeetingMode, BlurValues } from '../../types';
+
+const BLUR_OPTIONS = [
```

Comment:
Can we move this to `constants/index.ts` in the meeting app `src` directory?

Reply:
done